sqlglot/optimizer/qualify_columns.py

```diff
@@ -598,11 +598,12 @@ def _qualify_columns(
             if column_table:
                 column.set("table", column_table)
             elif (
-                resolver.schema.dialect == "bigquery"
+                resolver.schema.dialect in ("bigquery", "postgres")
```

Comment:
can you move these dialect checks into constants of the dialect?

Dialect.SUPPORTS...